doc/tutorials/ann/ann.txt

```diff
@@ -226,14 +228,52 @@ int main()
   // Train the model.
   model.Train(trainData, trainLabels);
 
-  // Use the Predict method to get the assignments.
-  arma::mat assignments;
-  model.Predict(trainData, assignments);
+  // Use the Predict method to get the predictions.
+  arma::mat predictionTemp;
+  model.Predict(testData, predictionTemp);
+
+  /* 
+    Since the predictionsTemp is of dimensions (3 x number_of_data_points) 
+    with continuous values, we first need to reduce it to a dimension of 
+    (1 x number_of_data_points) with scalar values, to be able to compare with
+    testLabels.
+
+    The first step towards doing this is to create a matrix of zeros with the 
+    desired dimensions (1 x number_of_data_points).
+  */
+  arma::mat prediction = arma::zeros<arma::mat>(1, predictionTemp.n_cols);
+
+  // Find index of max prediction for each data point and store in "prediction"
+  for (size_t i = 0; i < predictionTemp.n_cols; ++i)
+  {
+    // we add 1 to the max index, so that it matches the actual test labels.
+    prediction(i) = arma::as_scalar(arma::find(
+        arma::max(predictionTemp.col(i)) == predictionTemp.col(i), 1)) + 1;
+  }
+
+  /* 
+    Compute the error between predictions and testLabels, 
+    now that we have the desired predictions.
+  */
+  size_t error = 0;
+  for (size_t i = 0; i < testData.n_cols; i++)
+  {
+    if (int(arma::as_scalar(prediction.col(i))) == int(arma::as_scalar(testLabels.col(i))))
+    {
+        error++;
+    }
+  } 
+  double classificationError = 1 - double(error) / testData.n_cols;
+
+  // Print out the classification error for the testing dataset.
+  cout << "Classification Error for the Test set: " << classificationError << endl;
```

Comment:
There's no `using namespace std`, so we should either add that or change this to be `std::cout` and `std::endl`. :+1:

Reply:
Changed it to the latter option.